signup/signup.php

```diff
@@ -298,9 +297,15 @@ class="dcf-input-select"
 
 			<div appointment-picker class="dcf-mb-5"></div>
 
+			<div class="dcf-input-checkbox">
+				<input id="agree-to-virtual-preparation-checkbox" type="checkbox" ng-model="agreeToVirtualPreparationCheckbox.checked" value="false">
+				<label for="agree-to-virtual-preparation-checkbox">I agree to have my tax return prepared virtually. See <a href="https://www.irs.gov/pub/irs-pdf/f14446.pdf" target="_blank">Form 14446 (Virtual VITA/TCE Taxpayer Consent)</a></label>
```

Comment:
Any reason for using this version of the form rather than the one that we have on the server with the fields already set up?

Reply:
Tbh I just didn't want to copy the method for downloading the file over to this. What are your thoughts? The link Katie sent me was just to the IRS website, she never mentioned re-using the one that we have the clients filled out

Reply:
Makes sense. I'm fine with it how it is, but I was just curious if there was a reason for it being different.